docs/tool/js/views/map-view.js

```diff
@@ -994,5 +995,27 @@
                     'filter': ['==', 'nlihc_id', data]
                 });
             }
+        },
+        zoomToFilteredProjects: function(msg, data){
+            if (getState().filteredProjectsAvailable.length < 2 ) {
```

Comment:
@ostermanj what do you think about removing this condition? If we run this including the first time, this helps resolve the issue related to Ward 8 being hidden from view - the current initial load state cuts off a tiny bit. This is less of an issue now that we have a smaller legend, but depending what we end up with for the layers legend we can use the padding of this function to make sure that none of the project dots are ever hidden  by the legend.